aws/lambda/nodejs-graphql/waypoint.hcl

```diff
@@ -0,0 +1,31 @@
+project = "waypoint-apollo-lambda"
+
+app "waypoint-apollo-lambda" {
+  build {
+    use "docker" {
+      buildkit = true
+      platform = "amd64"
+    }
+
+    registry {
+      use "aws-ecr" {
+        region     = "us-east-1"
+        repository = "waypoint-apollo-lambda"
```

Comment:
Should we make this more "generic"? I imagine anyone who tries this is going to have to change it anyway. Maybe it could be an input var?

Reply:
Do you have an example of what this looks like? Or can you point me to docs? 🙏